src/index/IndexImpl.cpp

```diff
@@ -896,8 +896,7 @@ void IndexImpl::readConfiguration() {
   f >> configurationJson_;
   if (configurationJson_.find("git-hash") != configurationJson_.end()) {
     LOG(INFO) << "The git hash used to build this index was "
-              << configurationJson_["git-hash"].get<std::string_view>().substr(
-                     0, 6)
+              << configurationJson_["git-hash"].get<std::string_view>()
```

Comment:
Can you drop the `.get<...>` (I think JSON has an `operator<<`, it was just because of the `substr()` that you needed it.
```suggestion
              << configurationJson_["git-hash"]
```